atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/PostgresConnectionConfig.java

```diff
@@ -70,6 +73,10 @@ public Properties getHikariProperties() {
         props.setProperty("connectTimeout", Integer.toString(getConnectionTimeoutSeconds()));
         props.setProperty("loginTimeout", Integer.toString(getConnectionTimeoutSeconds()));
 
+        if (getDefaultRowFetchSize().isPresent()) {
+            props.setProperty("defaultRowFetchSize", getDefaultRowFetchSize().get().toString());
```

Comment:
See above about consistent naming, tests, and potential considerations about increasing the global default fetch size.

worth a comment linking to https://jdbc.postgresql.org/documentation/head/connect.html